specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/ComputerVision.json

```diff
@@ -1554,32 +1555,14 @@
         "format": "file"
       }
     },
-    "AzureRegion": {
-      "name": "AzureRegion",
-      "description": "Supported Azure regions for Cognitive Services endpoints",
+    "Endpoint": {
+      "name": "Endpoint",
+      "description": "Supported Cognitive Services endpoints",
```

Comment:
This is fine for now but would be great if there is a URL in here pointing users to a doc where he/she would be able to get the endpoint.